indico/modules/rb/models/reservations.py

```diff
@@ -604,6 +610,10 @@ def modify(self, data, user):
                 if field == 'booked_for_user':
                     old = self.booked_for_name
                     new = self.booked_for_name = data[field].full_name
+                # If notes are being updated then we don't bother updating the edit log with this
+                if field == 'notes':
+                    setattr(self, field, data[field])
+                    continue
                 # Apply the change
                 setattr(self, field, data[field])
```

Comment:
Might be simpler like this?
```suggestion
                # Apply the change
                setattr(self, field, data[field])
                # Don't record changes to notes
                if field == 'notes':
                    continue
```

Reply:
Or further down below: 

```python
if field == 'notes':
    continue
# Record change for history entry
```

Reply:
Nice :)